toolboxes/mri_image/hoMRImage.h

```diff
@@ -192,4 +189,16 @@ namespace Gadgetron
     };
 }
 
+template<class T, unsigned int D>
+void Gadgetron::Core::IO::read(std::istream &stream, Gadgetron::hoMRImage<T, D> &array);
```

Comment:
Again here especially, very confusing with the `array` variable name.

Reply:
fixed